packages/foundry/contracts/test/oracles/ChainlinkUSDMultiOracle.t.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.14;
+
+import "forge-std/src/Test.sol";
+import "@yield-protocol/utils-v2/contracts/token/ERC20.sol";
+import "../../oracles/chainlink/ChainlinkUSDMultiOracle.sol";
+import "../../oracles/chainlink/ChainlinkL2USDMultiOracle.sol";
+import "../../oracles/chainlink/AggregatorV3Interface.sol";
+import "../../mocks/oracles/chainlink/FlagsInterfaceMock.sol";
+import "../../mocks/oracles/chainlink/ChainlinkAggregatorV3MockEx.sol";
+import "../utils/TestConstants.sol";
+
+contract ChainlinkUSDMultiOracleTest is Test, TestConstants {
+    ChainlinkUSDMultiOracle public oracleL1;
+    ChainlinkL2USDMultiOracle public oracleL2;
+    FlagsInterfaceMock public flagsL2;
+    ChainlinkAggregatorV3MockEx public aggregator;
+    AggregatorV3Interface daiUsdAggregator = AggregatorV3Interface(0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9);
+    AggregatorV3Interface fraxUsdAggregator = AggregatorV3Interface(0xB9E1E3A9feFf48998E45Fa90847ed4D467E8BcfD);
+
+    address public dai = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public frax = 0x853d955aCEf822Db058eb8505911ED77F175b99e;
+
+    function setUp() public {
+        oracleL1 = new ChainlinkUSDMultiOracle();
+        flagsL2 = new FlagsInterfaceMock();
+        oracleL2 = new ChainlinkL2USDMultiOracle(flagsL2);
+        aggregator = new ChainlinkAggregatorV3MockEx(18);   // for testing non-8-digit source
+
+        vm.createSelectFork('mainnet', 15044600);
```

Comment:
Nice, I didn't know this was the way to do tests on mainnet forks.

Reply:
Previously, you had to run `forge test` with the `--fork-url` option if you wanted to use a mainnet fork and the `--fork-block-number` option if you didn't want to fork from the latest block. And all of your tests would uniformly be using that block. But now with the new cheatcodes, you can fork on a per-test basis where each one can fork from a different block and you no longer have to pass in those options. The `'mainnet'` is just an alias of the rpc url specified in `foundry.toml` and we're currently assigning through an environment variable.